samples/tutorial/Tutorial5.md

```diff
@@ -75,9 +75,7 @@ object WelcomeWorkflow : StatefulWorkflow<Unit, State, LoggedIn, WelcomeScreen>(
   }
```

Comment:
looks like more needs to be changed here. This is still using `name` rather than `username`

Reply:
Yes @steve-the-edwards you are right, updated with `username` just now. I also forget to update line `323`, it is now updated as well. Thanks!